src/bun.js/bindings/KeyObject.cpp

```diff
@@ -88,6 +88,12 @@ JSC_DECLARE_HOST_FUNCTION(KeyObject__privateDecrypt);
 
 namespace WebCore {
 
+/// Encodes `undefined` if `key` is `nullptr`.
+JSC::EncodedJSValue encodeKey(const std::optional<JSCryptoKey*> key)
```

Comment:
I would choose another name here just to add what is being encoded and to what is being encoded example: encodeKeyObjectToJS